src/main/java/org/simulator/examples/FBAExample.java

```diff
@@ -52,12 +52,12 @@
  * @author Shalin Shah
```

Comment:
Remove author information. Are we having author information on files in general? I find this unnecessary. All changes are tracked in git. So very clear who did what. 
Please ignore this if author information exists everywhere.

Reply:
I found it in most of the classes, so am not removing it.